docs/products/redis/concepts/overview.rst

```diff
@@ -0,0 +1,43 @@
+Aiven for Redis®* overview
+===========================
+
+Aiven for Redis®* is a fully managed **in-memory NoSQL database**, deployable in the cloud of your choice which can help you store and access data quickly and efficiently.
+
+Why Redis®?
+-----------
+
+Redis® is an open source, in-memory NoSQL datastore that serves as a fast data store, cache, or lightweight message broker. 
+
+With Aiven, you can leverage the power of Redis to improve the performance of your applications by easily setting up high-performance data caching. Additionally, Redis can be integrated seamlessly into your observability stack for purposes such as logging and monitoring.
+
+Redis offers a wide range of data structures, including strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes, and streams.
+
+Features and benefits of Aiven for Redis®*
+-------------------------------------------
+
+Aiven for Redis has many features that make it easy and stress-free to use:
+
+* **Managed service:** Aiven for Redis is fully managed, so you don't have to worry about setup, management, or updates. Aiven provides tools and integrations to help you easily use Redis in your data pipelines.
+
+* **Fast and easy deployment:** Aiven for Redis provides production-ready Redis clusters within a few minutes. You can deploy Redis to the cloud of your choice from 5 public clouds and over 100 regions. Aiven uses high-performance clusters with carefully selected instance types and storage options for top-notch performance. A Bring-your-own-account (BYOA) deployment model is available for strict control requirements.
```

Comment:
Shall we use Redis service instead? 

Redis Cluster is also the name of the plugin which RedisLab uses to support horizontal scaling. Tho w this context, it is understandable that this isnt "Redis Cluster".

Reply:
Changed this to Redis service.